Production/test/condorSub/makeDicts.py

```diff
@@ -0,0 +1,66 @@
+import os, glob, fnmatch
+from optparse import OptionParser
+
+meta_dict = {
+    "diboson" : ('WGJets','WW','WZ','ZG','ZZ'),
+    "dyjets"  : ('DYJetsToLL',),
+    "gjets"   : ('GJets',),
+    "qcd"     : ('QCD_HT',),
+    "qcd_pt"  : ('QCD_Pt',),
+    "signal"  : ('SMS','RPV','StealthSYY','StealthSHH'),
+    "singlet" : ('ST_s-channel','ST_t-channel','ST_tW','tZq'),
+    "ttbar"   : ('TTJets','TT_Tune'),
+    "tth"     : ('TTZTo','TTWJets','TTGJets','TTGamma','TTHH','TTTT','TTTW','TTWH','TTWW','TTWZ','TTZH','TTZZ','ttHJet'),
+    "wjets"   : ('WJetsToLNu',),
+    "zjets"   : ('ZJetsToNuNu',),
+}
+
+def main(args):
+    '''
+    Example usage:
+    python makeDicts.py -o tmp/
+    '''
+
+    # Read parameters
+    parser = OptionParser()
+    parser.add_option("-b", "--base",          dest="base",     default=os.environ["CMSSW_BASE"]+"/src/TreeMaker/Production/python/", help="Base path to the python folder containing the file lists (default = %(default)s")
+    parser.add_option("-d", "--debug",         dest="debug",    default=False, action="store_true",                                   help="Print extra debugging information (default = %(default)s)")
+    parser.add_option("-e", "--era",           dest="era",      default='RunIIFall17MiniAODv2',                                       help="The era used in the naming of the ouput dictionaries (default = %(default)s)")
+    parser.add_option("-l", "--location",      dest="location", default='RunIIFall17MiniAODv2/',                                      help="Location of the file lists which will be added to the dictionary (default = %(default)s)")
+    parser.add_option("-o", "--output_folder", dest="ofolder",  default="./",                                                         help="Put the output files in the specified folder (default = %(default)s)")
+    parser.add_option("-s", "--sig_senario",   dest="sigsen",   default='Fall17sig',                                                  help="The scenario used for the signal samples (default = %(default)s)")
```

Comment:
fix spelling of scenario (also in next arg)

Reply:
Fixed